test/e2e/workflow/nmstate_test.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 var _ = Describe("NMState", func() {
-	nmstateVersion := "v0.67.0"
+	nmstateVersion := "v0.70.1"
```

Comment:
Can we discover this number from github release or is not worth it ?

Reply:
I'm not sure if it would always help, since the container image we pull for operator is latest anyway
see https://github.com/kubevirt/cluster-network-addons-operator/pull/1339/files#diff-b523712def10219cc1b79738fa982d92baf4c16bcc6a7a2a26941de04b1e0b1aR70
and
https://github.com/kubevirt/cluster-network-addons-operator/pull/1339/files#diff-b523712def10219cc1b79738fa982d92baf4c16bcc6a7a2a26941de04b1e0b1aR123

Reply:
Well, we wouldn't need to update this, but only make sure we cut a release in knmstate, when making such changes